src/dbspro/rules.smk

```diff
@@ -9,15 +11,18 @@ abc_len = list(map(len, abc['Barcode-sequence']))[0]    # Assumes that all ABC a
 abc_umi_len = abc_len + config["umi_len"]
 dbs = "N"*config["dbs_len"]
 
+rule all:
```

Comment:
I'd just add a comment here to explain what this does since it isn't super intuitive from the keywords, especially since future people working on this pipeline probably won't have too much experience in snakemake.

Reply:
That is true I will definitely add that.